src/libcore/macros.rs

```diff
@@ -229,14 +229,26 @@ macro_rules! try {
     })
 }
 
-/// Use the `format!` syntax to write data into a buffer.
+/// Calls `write_fmt` function on a writer
 ///
-/// This macro is typically used with a buffer of `&mut `[`Write`][write].
+/// This macro takes an implementer of [`std::fmt::Write`][fmt_write] or
+/// [`std::io::Write`][io_write] trait, a precompiled format string, and a list of arguments.
+///
+/// Implementors of the `Write` trait are sometimes called 'writers'.
```

Comment:
Not sure which is correct, but on line 237 we have "Implementors" (with an O) and on line 234 "implementer" (with an E).